india_compliance/gst_india/utils/gstr_1/sections/_shared.py

```
@@ -20,6 +20,20 @@
 from india_compliance.gst_returns.fields.gstr1 import ItemField as item
 from india_compliance.gst_returns.fields.gstr1 import RawField as raw
 
+# generic steps live with the pure package; re-exported so sections keep one import
+from india_compliance.gst_returns.steps import add_item_totals as add_item_totals
```

Flake8 (7.3.0)

[error] 'india_compliance.gst_returns.steps.add_item_totals' imported but unused

(F401)